ci/cloudbuild/builds/coverage.sh

```diff
@@ -84,14 +84,15 @@ io::log "Flags: ${codecov_args[*]}"
 TIMEFORMAT="==> 🕑 codecov.io upload done in %R seconds"
 time {
   # Verifies the codecov bash uploader before executing it.
-  sha256sum="d6aa3207c4908d123bd8af62ec0538e3f2b9f257c3de62fad4e29cd3b59b41d9"
+  curl -sSL -o /var/tmp/codecov https://uploader.codecov.io/v0.2.3/linux/codecov
+  sha256sum="648b599397548e4bb92429eec6391374c2cbb0edb835e3b3f03d4281c011f401"
   codecov_url="https://raw.githubusercontent.com/codecov/codecov-bash/1b4b96ac38946b20043b3ca3bad88d95462259b6/codecov"
   codecov_script="$(curl -s "${codecov_url}")"
```

Comment:
`codecov_url` and `codecov_script` look dead now.

Reply:
Removed